### DISCUSSION TITLE
Not recognising ASTAP Platesolver star index

### DISCUSSION THREAD

**Discussion by mariusvr** [mworion/MountWizzard4#345] [Q&A] [answered]
Hi

I have ASTAP installed and it solves plates using the program itself with the D20 database.

However when I select it in mountwizzard4 settings, the folder application goes green, but the folder star index stays red.
![astap_error](https://github.com/mworion/MountWizzard4/assets/1148010/45924302-8978-4040-8880-e1138b521c6b)

Is there anything else that the program requires? I am on windows so I believe I can jot use watney as it requires astronomy.

Thanks! Marius

**Comment by mworion** [comment, marked as answer]
Marius,
no nothing to add. Please check if the index files are present. MW4 also counts the number of files, so the index has to be correct. In any other case please post the log that I could check what's going wrong.
Michel

**Comment by mariusvr** [comment]
Thank you for the very fast answer and making such amazing software available to us. 

I found the statistic screen and download the extra star libraries. Now I get a green light, so issue solved

**Comment by LpHiX** [comment]
Hi @mworion, I've noticed that the number of files for D20 seem to be wrong, the installer gives 1476 files but according to: https://github.com/mworion/MountWizzard4/blob/master/mw4/logic/plateSolve/astap.py the code thinks there should only be 1169, so my D20 never worked. D05 worked so I believe this is the issue

**Comment by mworion** [comment]
Martin,

thanks for the hint, I will check and correct this.

Michel

Gesendet mit der Telekom Mail App
<http://www.t-online.de/service/redir/emailmobilapp_ios_smartphone_footerlink.htm>

-----Original-Nachricht-----
Von: Martin Leung ***@***.***
***@***.***> >
Betreff: Re: [mworion/MountWizzard4] Not recognising ASTAP Platesolver star 
index (Discussion #345)
Datum: 25.07.2025, 01:32 Uhr
An: mworion/MountWizzard4 ***@***.***
***@***.***> >
CC: Michael ***@***.***
***@***.***> >, Mention <
***@***.*** ***@***.***> >

Hi @mworion <https://github.com/mworion> , I've noticed that the number of 
files for D20 seem to be wrong, the installer gives 1476 files but 
according to:
<https://github.com/mworion/MountWizzard4/blob/master/mw4/logic/plateSolve/astap.py> 
the code thinks there should only be 1169, so my D20 never worked. D05 
worked so I believe this is the issue

—
Reply to this email directly, view it on GitHub
<https://github.com/mworion/MountWizzard4/discussions/345#discussioncomment-13883231> 
, or unsubscribe
<https://github.com/notifications/unsubscribe-auth/ABSTMFGTWX7NUUQRCHAD4Q33KFUHFAVCNFSM6AAAAACCKF72HCVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTGOBYGMZDGMI> 
.
You are receiving this because you were mentioned.Message ID: 
***@***.***>
